eth_account/types.py

```diff
@@ -19,5 +20,6 @@
 PrivateKeyType = Union[Bytes32, int, HexStr, PrivateKey]
 
 AccessList = Sequence[Dict[str, Union[HexStr, Sequence[HexStr]]]]
+RLPStructuredAccessList = Tuple[Tuple[HexStr, Tuple[HexStr, ...]], ...]
```

Comment:
I wonder if this should be `Sequence` as well?

Reply:
updated